inscriptions/ISic001184.xml

```diff
@@ -190,15 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140668">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — —</supplied>οχλείας.
-                    <lb n="2"/>
+                    <lb n="1"/><gap reason="lost" extent ="unknown" unit="character"/><name><seg part="F">οχλείας</seg></name>.
```

Comment:
put the gap inside the name element but before the seg element (since it's the first part fo the name that's missing)